spec/models/machine_kit_compatibility_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+describe MachineKitCompatibility do
+  let(:blank_values) { [nil, ''] }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.